python/run-tests.py

```diff
@@ -37,9 +37,9 @@
 sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../dev/"))
 
 
-from sparktestsupport import SPARK_HOME  # noqa (suppress pep8 warnings)
-from sparktestsupport.shellutils import which, subprocess_check_output  # noqa
-from sparktestsupport.modules import all_modules  # noqa
+from sparktestsupport import SPARK_HOME  # noqa (suppress pep8 warnings) isort:skip
```

Comment:
it prevent these lines to be automatically moved on top of the sys.path.append. For me this is weird to have to modify the sys.path, probably because pyspark unit test are not executed inside a virtualenv